### PR TITLE
Start again (again)

### PR DESCRIPTION
**Pull request by BoogieMonster1O1**

### CODE REVIEW

File: src/main/java/io/github/hydos/holotext/HoloText.java [hunk 1]
```diff
@@ -4,58 +4,75 @@
...
+import net.fabricmc.fabric.api.event.lifecycle.v1.ServerEntityEvents;
 import net.fabricmc.loader.api.FabricLoader;
 
 public class HoloText implements ModInitializer {
 	public static final Logger LOGGER = LogManager.getLogger();
+	public static Config CONFIG;
 	private static final Path CONFIG_PATH = FabricLoader.getInstance().getConfigDir().resolve("holotext.json5");
 	private static final Jankson JANKSON = Jankson.builder().build();
-	private static final Codec<List<Entry>> ENTRIES_CODEC = Codec.list(Entry.CODEC);
+	private static final Consumer<String> PRINT_TO_STDERR = System.err::println;
```
Comment (hYdos) [inline src/main/java/io/github/hydos/holotext/HoloText.java:41]: how lazy can you be

File: src/main/java/io/github/hydos/holotext/config/Config.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package io.github.hydos.holotext.config;
+
+import java.util.Map;
+
+import com.google.common.collect.HashBiMap;
+import com.mojang.serialization.Codec;
+import com.mojang.serialization.codecs.RecordCodecBuilder;
+
+public class Config {
```
Comment (hYdos) [inline src/main/java/io/github/hydos/holotext/config/Config.java]: Codecs are the bane of my existence so i hope you got this right

File: src/main/java/io/github/hydos/holotext/mixin/MixinMinecraftServer.java [hunk 3]
```diff
@@ -12,6 +13,6 @@
 public class MixinMinecraftServer {
     @Inject(at = @At("RETURN"), method = "createCommandManager")
     public void intercept(CallbackInfoReturnable<CommandManager> infoIn) {
-
+        infoIn.getReturnValue().registerCommand(new CommandHolo());
```
Comment (hYdos) [inline src/main/java/io/github/hydos/holotext/mixin/MixinMinecraftServer.java]: we should just get commands-v1 working properly :p

**Review by hYdos — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/hydos/holotext/mixin/MixinMinecraftServer.java (hunk 3 above)
Comment (BoogieMonster1O1) [reply src/main/java/io/github/hydos/holotext/mixin/MixinMinecraftServer.java]: commands v1 doesn't add it to /help

File: src/main/java/io/github/hydos/holotext/HoloText.java (hunk 1 above)
Comment (BoogieMonster1O1) [reply src/main/java/io/github/hydos/holotext/HoloText.java:41]: is that badd

File: src/main/java/io/github/hydos/holotext/mixin/MixinMinecraftServer.java (hunk 3 above)
Comment (hYdos) [reply src/main/java/io/github/hydos/holotext/mixin/MixinMinecraftServer.java]: make it do it

Comment (BoogieMonster1O1) [reply src/main/java/io/github/hydos/holotext/mixin/MixinMinecraftServer.java]: concern
i don't think it's possible without breaking api

Comment (hYdos) [reply src/main/java/io/github/hydos/holotext/mixin/MixinMinecraftServer.java]: no one uses v1 yet

Comment (hYdos) [reply src/main/java/io/github/hydos/holotext/mixin/MixinMinecraftServer.java]: (properly)

Comment (BoogieMonster1O1) [reply src/main/java/io/github/hydos/holotext/mixin/MixinMinecraftServer.java]: explain that to @shedaniel 
do you think anyone used those seperate registry events? no
did I still have to keep them? yes

Comment (hYdos) [reply src/main/java/io/github/hydos/holotext/mixin/MixinMinecraftServer.java]: huh

File: src/main/java/io/github/hydos/holotext/HoloText.java (hunk 1 above)
Comment (hYdos) [reply src/main/java/io/github/hydos/holotext/HoloText.java:41]: yes. fix or remove

Comment (BoogieMonster1O1) [reply src/main/java/io/github/hydos/holotext/HoloText.java:41]: no

Comment (BoogieMonster1O1) [reply src/main/java/io/github/hydos/holotext/HoloText.java:41]: i leik it

Comment (hYdos) [reply src/main/java/io/github/hydos/holotext/HoloText.java:41]: guess i wont merge :p

Comment (BoogieMonster1O1) [reply src/main/java/io/github/hydos/holotext/HoloText.java:41]: concern. explain why

Comment (hYdos) [reply src/main/java/io/github/hydos/holotext/HoloText.java:41]: because thats more jank than jankson

Comment (hYdos) [reply src/main/java/io/github/hydos/holotext/HoloText.java:41]: and dfu

Comment (BoogieMonster1O1) [reply src/main/java/io/github/hydos/holotext/HoloText.java:41]: it's just a consumer

**Comment by BoogieMonster1O1** [comment]
now serializes the data directly to the holotext.
removed a bunch of classes too

**Review by hYdos — APPROVED** [review]
good

File: gradle.properties [hunk 4]
```diff
@@ -4,7 +4,7 @@ org.gradle.jvmargs=-Xmx2G -XstartOnFirstThread
 # Fabric Properties
 	# check these on https://fabricmc.net/use
 	minecraft_version = 1.8.9
-	yarn_mappings = 1.8.9+build.202009070256
+	yarn_mappings = 1.8.9+build.202009230433
```
Comment (hYdos) [inline gradle.properties:7]: image

**Review by hYdos — CHANGES_REQUESTED** [review]

File: gradle.properties (hunk 4 above)
Comment (BoogieMonster1O1) [reply gradle.properties:7]: that will require replacing every single entity method in the mod with `method_xxxx`

Comment (hYdos) [reply gradle.properties:7]: lmao

**Pull request merged**